src/helpers/TestOAuthServer.js

```diff
@@ -0,0 +1,46 @@
+// test OAuth server that supports only client_credentials grant type
+// with a fixed set of client id and secret values configured in auth-settings.js
+
+const basicAuth = require('express-basic-auth');
+const crypto = require('crypto');
+const router = require('express').Router();
+
+const Cache = require('ttl');
+const { oauthServer } = require('../../auth-settings');
+
+const cache = new Cache({
+  ttl: 3600 * 1000
+});
```

Comment:
I think we've used this Cache declaration twice now?  Should we abstract to a helper function and/or at least move the default TTL value to a shared constant file?